<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameCopyDialog</class>
 <widget class="QDialog" name="GameCopyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copy &amp; Convert Parameters</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_file">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string>Enter a specific destination for this game</string>
         </property>
         <property name="text">
          <string>Enter a Filename</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_destOther">
           <property name="toolTip">
            <string>Enter a specific destination for this game</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_destOther">
           <property name="toolTip">
            <string>Enter a specific destination for this game</string>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_partition">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>153</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Use a known partition as the destination</string>
       </property>
       <property name="title">
        <string>Usa an Existing Partition</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_dest">
          <property name="minimumSize">
           <size>
            <width>116</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>/&lt;Enter a Directory&gt;</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_partitionisWBFS">
          <property name="toolTip">
           <string>Treat the destination as a WBFS partition</string>
          </property>
          <property name="text">
           <string>Partition has WBFS filesystem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_naming">
          <property name="text">
           <string>Naming Scheme:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_esc">
          <property name="toolTip">
           <string>How should the games me named when written to the destination?</string>
          </property>
          <item>
           <property name="text">
            <string>/ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/GameName.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/GameName ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/GameName_ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/ID_GameName.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/GameName [ID]/ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/GameName_[ID]/ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/ID_GameName/ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/FileName.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/FileName [ID].ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/FileName_[ID].ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/ID_FileName.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/FileName [ID]/ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/FileName_[ID]/ID.ext</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/ID_FileName/ID.ext</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_split">
       <property name="toolTip">
        <string>Check this box to split games that are larger than 4GB</string>
       </property>
       <property name="text">
        <string>Split files at 4GB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_overwrite">
       <property name="toolTip">
        <string>Check this box to overwrite existing files without warning</string>
       </property>
       <property name="text">
        <string>Overwrite Existing files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_partitions">
       <property name="toolTip">
        <string>Check the box for each type of partition you would like to remain in teh copied game</string>
       </property>
       <property name="title">
        <string>Keep Partitions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QCheckBox" name="checkBox_part_update">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_part_data">
          <property name="text">
           <string>Game</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_part_installer">
          <property name="text">
           <string>Channel installer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_part_id">
          <property name="text">
           <string>VC Trials</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_container">
         <property name="text">
          <string>Container:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_container">
         <property name="toolTip">
          <string>What type of container should hold the game?</string>
         </property>
         <item>
          <property name="text">
           <string>wbfs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>iso</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fst</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wia</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>wdf</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ciso</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label_example">
     <property name="toolTip">
      <string>Example Output filename</string>
     </property>
     <property name="text">
      <string>ExampleOutput</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="minimumSize">
      <size>
       <width>344</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_blank"/>
     <widget class="QWidget" name="page_iso">
      <widget class="QLabel" name="label_iso">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Iso Options</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_iso_trunc">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Truncate the output .iso to a smaller size</string>
       </property>
       <property name="text">
        <string>Truncate iso</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_fst">
      <widget class="QLabel" name="label_fst">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Fst Options</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_fst_sneek">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>171</width>
         <height>61</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Keep only needed files for use with SNEEK's DI module</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/sneek.png</normaloff>:/images/sneek.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>134</width>
         <height>47</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_wia">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_wia">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Wia Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_wia_presets">
         <property name="toolTip">
          <string>Use a preset compression setting</string>
         </property>
         <property name="title">
          <string>Preset Mode</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QComboBox" name="comboBox_wia_preset">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Fast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Good</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Best</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_wia_advanced">
         <property name="toolTip">
          <string>Manually enter compression options</string>
         </property>
         <property name="title">
          <string>Advanced</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_wia_method">
              <property name="text">
               <string>Method:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_wia_method">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purge</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bzip2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lzma</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Lzma2</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_wia_level">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>74</height>
             </size>
            </property>
            <property name="title">
             <string>Level</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="comboBox_wia_level">
               <item>
                <property name="text">
                 <string>Default</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_wia_chunk">
            <property name="title">
             <string>Chunk factor</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QSpinBox" name="spinBox_wia_chunk">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>20</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameCopyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameCopyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
